index.html

```diff
@@ -92,6 +92,8 @@ <h1 id="version-info-id">v1.0.34</h1>
           friends! <br />Supported by Curious Learning.
         </p>
       </div> -->
+      <!-- Replace the canvas with a div for feedback text -->
+      <div id="feedback-text" class="feedback-text">amazing</div>
```

Comment:
Why is amazing hard coded? What about the other text message, I believe there are words 'good' and 'great'

Reply:
@bernhardccodev i adjusted this. please check